src/app/components/profile/profile.component.ts

```diff
@@ -125,6 +125,12 @@ export class ProfileComponent implements OnInit {
    * Displays a Modal to update user details
    */
   updateUserDetails() {
+    const firstName = (this.user['first_name'] === '-') ? '' : this.user['first_name'];
+    const lastName = (this.user['last_name'] === '-') ? '' : this.user['last_name'];
+    const affiliation = (this.user['affiliation'] === '-') ? '' : this.user['affiliation'];
+    const google_scholar_url = (this.user['google_scholar_url'] === '-') ? '' : this.user['google_scholar_url'];
+    const github_url = (this.user['github_url'] === '-') ? '' : this.user['github_url'];
+    const linkedin_url = (this.user['linkedin_url'] === '-') ? '' : this.user['linkedin_url'];
```

Comment:
You need to camel-case rest 3 variables also. :smiley:

Reply:
googleScholarUrl or googlescholarUrl ?

Reply:
googleScholarUrl